Picker/ViewController.swift

```diff
@@ -8,33 +8,50 @@
 
 import UIKit
 
-class ViewController: UIViewController {
+class ViewController: UIViewController, UICollectionViewDataSource, UICollectionViewDelegateFlowLayout {
 
     @IBOutlet weak var collectionView: UICollectionView!
     
     override func viewDidLoad() {
-//        super.viewDidLoad()
+        super.viewDidLoad()
 //        let cardLayout = UICollectionViewFlowLayout()
 //        cardLayout.sectionInset = UIEdgeInsets(top: 30, left: 20, bottom: 50, right:20)
 //        cardLayout.itemSize = CGSize(width :236,height :367)
 //        cardLayout.minimumLineSpacing = 75
 //        cardLayout.scrollDirection = UICollectionViewScrollDirection.Horizontal
 //        collectionView = UICollectionView(frame: self.view.frame, collectionViewLayout: cardLayout)
-//        self.collectionView.dataSource = self
-//        self.collectionView.delegate = self
-//        self.collectionView.pagingEnabled = true
+        self.collectionView.dataSource = self
+        self.collectionView.delegate = self
+        self.collectionView.pagingEnabled = true
 //        collectionView.registerClass(CustomCellView.self, forCellWithReuseIdentifier: CustomCellView.cellIdentifier)
-//        collectionView.backgroundColor = UIColor.whiteColor()
-//        self.collectionView.showsHorizontalScrollIndicator = false
-//        self.view.addSubview(collectionView!)
+        collectionView.backgroundColor = UIColor.whiteColor()
+        self.collectionView.showsHorizontalScrollIndicator = false
+        self.view.addSubview(collectionView!)
```

Comment:
You don't need to do this. It is already added as a `subView` in the `Main.storyboard`